src/index.html

```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet" />
+    <link rel="preconnect" href="https://use.typekit.net" crossorigin />
+    <link href="https://use.typekit.net/fjg7yrm.css" rel="stylesheet" />
```

Comment:
In think we could refine this to only fetch the particular items we need in a similar way to what's done in the <head> section of [https://www.scottlogic.com/](https://www.scottlogic.com/) if page load times proved to be an issue.